api4/cloud.go

```diff
@@ -665,6 +665,15 @@ func handleCWSWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 			return
 		}
 		c.Logger.Info("Updated subscription from webhook event")
+	case model.EventTypeTriggerDelinquencyEmail:
+		var emailToTrigger model.DelinquencyEmail
+		if event.DelinquencyEmail != nil {
+			emailToTrigger = model.DelinquencyEmail(event.DelinquencyEmail.EmailToTrigger)
+		}
```

Comment:
Should we do an early return if `event.DelinquencyEmail` is `nil`?

Reply:
Added